restapi/delete-blocks.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Delete blocks of data'
```

Comment:
I would expect the details of this endpoint to be in the Dataset endpoints section. What is the reason this does not feature in Swagger definition?

Reply:
We agreed that this endpoint is not ready for GA.